Gemfile

```diff
@@ -10,9 +10,13 @@ gem 'jquery-rails'
 gem 'turbolinks'
 gem 'jquery-turbolinks'
 gem 'jbuilder', '~> 2.0'
+gem 'activeadmin', github: 'gregbell/active_admin'
```

Comment:
Waqas bhai It was version problem. Gem has a pre version while their github has the stable one. That's why i included it from github to avoid errors. Rabnawaz did include the new version but he faced alot of trouble during setting up working.